dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/client/injection/ServiceRefObjectFactory.java

```diff
@@ -112,7 +111,6 @@ protected void deActivate(ComponentContext cCtx) {
     protected void setSecurityConfigurationService(ServiceReference<JaxWsSecurityConfigurationService> serviceRef) {
         securityConfigSR.setReference(serviceRef);
         LibertyProviderImpl.setSecurityConfigService(securityConfigSR);
-        LibertyHTTPTransportFactory.setSecurityConfigService(securityConfigSR);
```

Comment:
See the above comment about removing Liberty's extensions of CXF's Transport HTTP Module